src/zenml/models/v2/core/artifact_version.py

```diff
@@ -167,6 +174,11 @@ class ArtifactVersionResponseMetadata(WorkspaceScopedResponseMetadata):
 class ArtifactVersionResponseResources(WorkspaceScopedResponseResources):
     """Class for all resource models associated with the artifact version entity."""
 
+    pipeline_run_ids: Optional[List[UUID]] = Field(
```

Comment:
I am a bit in-between about adding this here. When we created the concept of `Resources`, the idea was that this model would only feature other zenml models or a page of zenml models (not a list of IDs). However, since we are talking about pipeline runs here, it is a bit hard to decide on this one. I will leave it up to you.

Reply:
I feel that you are right here - I removed all properties, which were not originally requested